nvcf/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG BASE_IMAGE
+ARG BASE_TAG
+FROM ${BASE_IMAGE}:${BASE_TAG}
+
+ARG NGC_KEY
+ENV NGC_API_KEY=${NGC_KEY}
```

Comment:
I have not seen NGC_KEY used in any other automation, think this should stay as NGC_API_KEY unless you have seen NGC_KEY elsewhere?

Reply:
We should not be setting keys here, you can set these via CLI or the UI.

Reply:
I will move the API key from docker image and set it during function creation. Regarding the name, I didn't use NGC_API_KEY b/c it's actually has to be personal key, instead of the api key. But I agreed that we should stick to the NGC_API_KEY name convention. will change it